Assets/VRTK/Examples/Resources/Scripts/ScaleInteractableObject.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace VRTK.Examples
```

Comment:
Because that script is part of the examples `VRTK.Examples` is the proper namespace I think. Referencing other VRTK stuff can probably be simplified like @10FingerArmy meant by adding `using VRTK;` below.